fpdf/html.py

```diff
@@ -21,9 +48,19 @@ def px2mm(px):
 def hex2dec(color="#000000"):
     if not color:
         return None
-    r = int(color[1:3], 16)
-    g = int(color[3:5], 16)
-    b = int(color[5:7], 16)
+
+    # Checks if color is a name and gets the hex value
+    hexcolor = COLOR_DICT.get(color.lower(), color)
+
+    if len(hexcolor) == 4:
```

Comment:
Why is this "if" needed?

Reply:
because html hex codes can also be specified as `#123` as short for `#112233`